src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+$day-height: 100px;
```

Comment:
Consider using a more descriptive variable name for the height of the calendar cells, such as $calendar-cell-height.